tests/nnfw_api/src/one_op_tests/Softmax.cc

```diff
@@ -62,6 +62,23 @@ TEST_P(SoftmaxVariation, Test)
   SUCCEED();
 }
 
+TEST_F(GenModelTest, OneOp_Softmax)
```

Comment:
Could you please add negative test for beta != 1 ?

Reply:
Could I commit with separated new PR for neg tc?

Reply:
@chunseoklee PTAL.
I have add negative test case.